Resources/Server Info/Guidebook/Chef/pitzah.xml

```diff
@@ -96,4 +103,102 @@
 <GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
 <GuideEntityEmbed Entity="FoodPizzaDonkpocket" Caption=""/>
 </Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Margherita Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Mozzarella
+- Cheese wedge
+- Ambrosia Vulgaris
+</Box>
+<Box Orientation="Vertical">
+- Dough
+- 2 Tomatoes
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
+<GuideEntityEmbed Entity="FoodMothPizzaMargherita" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Firecracker Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- 10u BBQ sauce
+- Cheese wedge
+- Baked Corn
+</Box>
+<Box Orientation="Vertical">
+- Dough
+- Chili pepper
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
+<GuideEntityEmbed Entity="FoodMothPizzaFirecracker" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Five-Cheese Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Dough
+- 2 Tomatoes
+- Mozzarella
+</Box>
+<Box Orientation="Vertical">
+- Cheese Wedge
+- Cheese Curds
+- Curd Cheese
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
+<GuideEntityEmbed Entity="FoodMothPizzaFiveCheese" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Pesto Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Dough
+- Mozzarella
+- Pesto
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
+<GuideEntityEmbed Entity="FoodMothPizzaPesto" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Garlic Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Dough
+- Butter
+</Box>
+<Box Orientation="Vertical">
+- Garlic
+- Ambrosia Vulgaris
+</Box>
```

Comment:
These two recipes might do with a merging that would result in a pizza equivalent of Aglio i Olio. Basically a Garlic Pizza Bread. In fact, toppings of those two are pretty much exactly a formula of Aglio i Olio split in two, and why keep garlic and pesto from each other is beyond me. And "light" pizzas like that are a perfectly valid food.

Reply:
Eh, I'd argue a Pesto Pizza and Garlic Pizza have different flavors.
I'd make the Pesto recipe use tomatoes and the Garlic use Mozzarella instead of Herbs to differentiate them a bit if that can work.

Reply:
See, Pesto is already everything the Garlic Pizza is because it's made with herbs, garlic, oil. At best, you can make the Garlic Pizza an alternate recipe, but they're both nearly identical in contents, except one's made with olivol and the other with butter. Normally, I'd be up for variety in preparing foods, except the cooking system is shit at it, so I lean on the side of debloating, and it's unnecessary here because olivol isn't hard to come by, so you don't need another recipe as a substitute.